src/Events/Event.php

```diff
@@ -20,8 +20,6 @@
 class Event implements \ArrayAccess, \IteratorAggregate, \Countable
 {
 
-	use \Kdyby\StrictObjects\Scream;
```

Comment:
I get that you removed them from classes that inherit from Nette, but why also these classes?

Reply:
Out of pure laziness, I guess :D First of all I removed it with global find-replace and second I'd have to fork and fix yet another package. I'm willing to do it if it really makes sense to do it.